threedi_models_and_simulations/widgets/simulation_wizard.py

```diff
@@ -681,6 +696,66 @@ def connect_signals(self):
         self.pb_upload_2d_laterals.clicked.connect(partial(self.load_csv, self.TYPE_2D))
         self.cb_2d_interpolate.stateChanged.connect(partial(self.interpolate_changed, self.TYPE_2D))
 
+    def setup_substance_concentrations(self):
+        if hasattr(self, "groupbox"):
+            self.groupbox.setParent(None)
+        if not self.substances:
+            return
+        substance_concentration_widget = SubstanceConcentrationsWidget(self.substances, self.handle_substance_timesteps)
+        self.groupbox = substance_concentration_widget.groupbox
```

Comment:
Instance attribute `groupbox `defined outside __init__. Please initialize it within __init__ first.